### PR TITLE
CASSANDRA-16980: Added September 2021 blog post on ApacheCon 2021

### PR DESCRIPTION
**Pull request by nonstopdtop**
- Blog post is titled "Join Cassandra at Apachecon 2021" about ApacheCon 2021
- Modified blog index page

### CODE REVIEW

**Review by ErickRamirezDS — APPROVED** [review]
@michaelsembwever has pushed it to staging and it looks good to me so it should be good to push to production.

I'll post an update on CASSANDRA-16980. Cheers! 🍻

**Comment by michaelsembwever** [comment]
merged with 5ab1567feb821e7b344fcac8dd8f262bde30b1c9

File: site-content/source/modules/ROOT/pages/blog/Join-Cassandra-at-ApacheCon-2021.adoc [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+* *Evolving Transactions in Apache Cassandra* — Benedict Elliot Smith, Apache Cassandra committer
+* *Designing Keys for NOSQL solutions* — Nikolai Kolesnikov, AWS
+* *Cassandra Data Migration with Dual Write Proxy* — German Eichberger, Microsoft Azure Data & AI
+* *Modeling Financial Data In Cassandra To Serve Real Time And Batch Workloads At Same Time* — Gokul Prabagaren, Capital One
+* *How Netflix Provisions Optimal Cloud Deployments of Cassandra* — Joey Lynch, Netflix
+* *Stargate.io, An OSS API Layer for your Cassandra* — Cedrick Lunven, DataStax
+* *Cassandra for giant 3D tables* — David North, CoreFiling
+
+At the end of the first track day on Tuesday, we’ll be hosting a Birds of Feather (BoF) session
+from 2030 to 2110 UTC with Melissa Logan hosting a discussion about Cassandra 4.0 between Patrick McFadin and Aaron Ploetz. We’ll dig deeper into the benefits, upgrade, and more. Bring your questions to this interactive discussion!
```
Comment (ossarga) [inline site-content/source/modules/ROOT/pages/blog/Join-Cassandra-at-ApacheCon-2021.adoc:32]: Should we fix the two times here so they consistent with the previous time further up?

The change would be

> ... from 20:30 to 21:10 UTC with Melissa Logan ...

**Review by ossarga — COMMENTED** [review]
Found two typos we may want to fix.

**Pull request closed without merging**